packages/elements/src/react/common/form/index.tsx

```diff
@@ -270,6 +273,29 @@ const useInput = ({
     ref.send({ type: 'FIELD.UPDATE', field: { name, value: initialValue } });
   }, [name, ref, initialValue]);
 
+  const [isSupported, setIsSupported] = React.useState(false);
```

Comment:
That name is too generic 😅 Further below I didn't know anymore what `isSupported` refers to

Reply:
I've now removed that state